### DISCUSSION TITLE
Feature request: Add F# support for Package Dependency Graph

### DISCUSSION THREAD

**Discussion by teo-tsirpanis** [community/community#21846] [New to GitHub]
<p>Regarding .NET support for displaying the dependencies of a package, there is support for C#, Visual Basic, and raw .nuspec files, but not for F# .fsproj files, as shown here:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/4c3a329766a6987e74e40939cc517fa225ee2957.png" data-download-href="/uploads/short-url/aSkYu9o5w6dtfNuuPBLy7XKUvyv.png?dl=1" title="Annotation 2019-06-07 131858.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12659251/181067107-0988715c-0e08-4f9c-8a49-fc1dc2855aa2.png" alt="Annotation 2019-06-07 131858.png" data-base62-sha1="aSkYu9o5w6dtfNuuPBLy7XKUvyv" width="690" height="371" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/4c3a329766a6987e74e40939cc517fa225ee2957_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Annotation 2019-06-07 131858.png</span><span class="informations">736×396 17 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Adding support for .fsproj files would seem easy, but an even better step is to add support for <a href="https://fsprojects.github.io/Paket/" rel="nofollow noopener">Paket</a> dependencies, as it is very popular among F# developers.</p>
<p>This feature would give feedback on F# developers to better understand who uses their repository, as well as protct them from <a href="https://snyk.io/vuln?type=nuget" rel="nofollow noopener">security vulnerabilities in NuGet packages</a>.</p>

**Comment by BelHFL** [comment, marked as answer]
<p>Hi <a class="mention" href="/teo-tsirpanis">@teo-tsirpanis</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Cheers!</p>

**Comment by teo-tsirpanis** [comment]
<p>Hello, I happened to see that dependencies are properly displayed for .fsproj files.</p>
<p>Good job. Thanks a lot! :v:t2:</p>
